terraform/main.tf

```diff
@@ -0,0 +1,58 @@
+data "azurerm_resource_group" "rg" {
+  name = var.resource_group_name
+}
+
+module "networking" {
+  source              = "./modules/networking"
+  vnet_name           = var.vnet_name
+  resource_group_name = data.azurerm_resource_group.rg.name
+  location            = var.location
+  address_space       = var.address_space
+  subnets = [
+    {
+      name   = var.subnet_name
+      prefix = var.subnet_prefix
+    }
```

Comment:
Aici vrem 2 subnet-uri, fiecare vm sa fie in subnet diferit ca sa fie relativ separat. Probabil trebuie de modificat si networking module si vm module ca sa poata functiona asa